src/TodoInfo.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import cn from 'classnames';
+import { Todo } from './types/Todo';
+
+interface Props {
+  todo: Todo;
+  selectedTodo: Todo | null;
+  selectTodo: (todoId: Todo | null) => void
+}
+
+export const TodoInfo: React.FC<Props> = ({
+  todo,
+  selectTodo,
+  selectedTodo,
+}) => {
+  return (
```

Comment:
you should return right away in this arrow function, as you don't have any logic implemented before the return statement
```suggestion
}) => (
```